src/vs/base/browser/markdownRenderer.ts

```diff
@@ -662,7 +664,14 @@ export function renderAsPlaintext(str: IMarkdownString | string, options?: {
 		value = `${value.substr(0, 100_000)}…`;
 	}
 
-	const html = marked.parse(value, { async: false, renderer: options?.includeCodeBlocksFences ? plainTextWithCodeBlocksRenderer.value : plainTextRenderer.value });
+	let renderer = plainTextRenderer.value;
+	if (options?.includeCodeBlocksFences) {
+		renderer = plainTextWithCodeBlocksRenderer.value;
+	} else if (options?.preserveFileLinks) {
```

Comment:
Using the API it would be surprising you can't set both `includeCodeBlocksFences` and `preserveFileLinks` can be set. Can we make it so both can be enabled?